api/events.go

```diff
@@ -0,0 +1,49 @@
+package api
+
+type Command struct {
+	Name             CommandName              `json:"name"`
+	AddBreakPoint    *AddBreakPointCommand    `json:"addBreakPoint"`
+	ClearBreakPoints *ClearBreakPointsCommand `json:"clearBreakPoints"`
+	Detach           *DetachCommand           `json:"detach"`
+	Kill             *KillCommand             `json:"kill"`
+}
+
+type CommandName string
```

Comment:
Just an idiom I picked up for enforcing constants... a poor man's enum, if you will. If you think it's pointless I don't mind using `string`.